### DISCUSSION TITLE
FTP record schedule setting are not preserved

### DISCUSSION THREAD

**Discussion by tomasjuchelka** [fwestenberg/reolink_dev#389] [Q&A] [answered]
Hi all,

I figured out that FTP recording schedule settings are not preserved, if I disable/enable FTP upload using this integration.
If I do it via app it works fine.
If I re-enable FTP upload in my HA, all setting for detection types are not set anymore.

I would like to enable FTP upload only when "not_home" and disable it when "home".
Or is there an easier way?
Thanks for help!
Tomas

**Comment by cpainchaud** [comment, marked as answer]
Hi please try v0.43

https://github.com/fwestenberg/reolink_dev/commit/5957790c62fae0027ff5a492628a526355c3b042

**Reply by tomasjuchelka** [reply]
Great, it works :)
